admin-site/src/components/HelloWorld.vue

```diff
@@ -3,13 +3,17 @@
   <img src="C:\Users\amydo\Downloads\login.jpg" alt="Login image">-->
   <div class="login">
     <div style="display: flex; justify-content: center; align-items: center; height: 10vh;">
-  <h1>Pluggable Login</h1>
+  <h1>Admin-site Login</h1>
     </div>
     <form>
       <div class="form-group">
         <label for="email">Email address</label>
         <input type="email" class="form-control" id="email" v-model="email" placeholder="Enter email">
       </div>
+      <div class="form-group">
+        <label for="password">Password</label>
+        <input type="password" class="form-control" id="password" v-model="password" placeholder="Enter password">
+      </div>
       <button type="submit" class="btn btn-primary" @click.prevent="submit">
```

Comment:
Lets add a `@click` action to this method and a method in the `script` section below that just console.logs the email and password as a placeholder for the minute